### ISSUE TITLE
Not working for typescript

### ISSUE THREAD

**Issue by ckarnell** [keith/investigate.vim#28]
I have added docs to Dash for typescript and restarted vim, but I keep seeing "No documentation for typescript" when running the command to access the docs from Dash. I've used this plugin for years without issue, so I'm assuming there's just a value in a hashmap somewhere for typescript that's missing? I haven't peeked at the code though. Any ideas on how to fix?

**Comment by keith** [comment]
Yep, will need to be added here https://github.com/keith/investigate.vim/blob/aef9332ba3cfc070fb59fd7a4ac82bae2b42cd7b/autoload/investigate/defaults.vim#L12-L44
